etc/docker/kubeflow/README.md

```diff
@@ -16,15 +16,18 @@ limitations under the License.
 {% endcomment %}
 -->
 
-### Elyra notebook container image for use with Kubeflow's Notebook Server
+### Elyra Docker Image - Kubeflow Notebooks
 
-This `Dockerfile` is used to build an Elyra notebook image that can be launched by [Kubeflow's Notebook Server](https://www.kubeflow.org/docs/components/notebooks/). Ready-to-use  container images are published on [Docker Hub](https://hub.docker.com/r/elyra/kf-notebook) and [quay.io](https://quay.io/repository/elyra/kf-notebook). Refer to [the documentation](https://elyra.readthedocs.io/en/latest/recipes/using-elyra-with-kubeflow-notebook-server.html) for details.
+This `Dockerfile` is used to build an Elyra notebook image that can be launched by [Kubeflow's Notebook Server](https://www.kubeflow.org/docs/components/notebooks/).
+
+Ready-to-use container images are published on [Docker Hub](https://hub.docker.com/r/elyra/kf-notebook) and [quay.io](https://quay.io/repository/elyra/kf-notebook). 
+Refer to [the documentation](https://elyra.readthedocs.io/en/latest/recipes/using-elyra-with-kubeflow-notebook-server.html) for details.
 
 #### Building a custom container image
 
 To build a custom version of this container image:
-- Clone this repository.
-- Update the requirements in `etc/docker/kubeflow/requirements.txt`.
-- Run `make kf-notebook-image` in the root directory of this repository.
-
-> The container image is automatically tagged with `elyra/kf-notebook:dev` and `quay.io/elyra/kf-notebook:dev`.
+1. Clone this repository
+2. Build the image using the `Dockerfile` in the `kubeflow/notebooks` directory:
```

Comment:
The directory reference appears to be invalid

```suggestion
2. Build the image using the `Dockerfile` in the `etc/docker/kubeflow` directory:
```

Reply:
Done in https://github.com/elyra-ai/elyra/pull/2344/commits/c5e17124f04ab5d2016ceb558cda1281e146ca12